.depcheckrc.yml

```diff
@@ -12,6 +12,7 @@ ignores:
   - '@metamask/create-release-branch'
   - 'depcheck'
   - 'eslint-interactive'
+  - 'rimraf'
```

Comment:
Only listed as a devDep in the root package.json, where it's used in a build script but not in any modules.